test/extends.test.js

```diff
@@ -17,7 +17,7 @@ describe("extend", function() {
       result.should.have.type("string");
 
       // verify the generated module imports the `include`d templates
-      result.should.match(/require\(\"twig\!a\.html\.twig\"\);/);
```

Comment:
For some reason when keeping the twig loader prefix, the webpack uses a different loader context for those files and doesn't use the query config that I have defined in my webpack config file. Any ideas on how to solve that?